File: maiworld-seller-interface/src/main/java/com/maiworld/seller/service/GroundService.java
package com.maiworld.seller.service;

import java.util.List;

import com.maiworld.pojo.TbGround;

import entity.PageResult;

/**
 * 培训场地详情
 * @author Administrator
 *
 */
public interface GroundService {

	/**
	 * 返回所有数据
	 * @return
	 */
	public List<TbGround> findAll();
	/**
	 * 分页
	 * @param pageNum显示当前页
	 * @param pageSize 每页显示记录数
	 * @return
	 */
	public PageResult findPage(int pageNum , int pageSize);
	/**
	 * 增加
	 * @param ground
	 */
	public void add(TbGround ground);
	
	/**
	 * 修改
	 * @param ground
	 */
	public void update(TbGround ground);
	
	/**
	 *	根据ID查询实体
	 * @param id
	 * @return
	 */
	public TbGround fingOne(Long id);
	
	/**
	 * 批量删除
	 * @param ids
	 */
	public void delete(Long[] ids);
	
	/**
	 * 查询+分页
	 * @param ground
	 * @param pageNum
	 * @param pageSize
	 * @return
	 */
	public PageResult findPage(TbGround  ground , int pageNum, int pageSize);
	
}

File: maiworld-seller-web/src/main/webapp/js/service/groundService.js
//服务层
		app.service('groundService' ,function($http){
    				this.findAll=function(){
    					return $http.get('../ground/findAll.do');
    				}
    				
    				this.findPage=function(page, rows){
    					return $http.get('../ground/findPage.do?page='+page+'&rows='+rows);
    				}
    				
    				this.findOne=function(id){
    					return $http.get('../ground/findOne.do?id='+id);
    				}
    				
    				this.add=function(entity){
    					return $http.post('../ground/add.do',entity);
    				}
    				
    				this.update=function(entity){
    					return $http.post('../ground/update.do',entity);
    				}
    				
    				this.dele=function(ids){
    					return $http.get('../ground/delete.do?ids='+ids);
    				}
    				
    				this.search=function(page, rows, searchEntity){
    					return $http.post('../ground/search.do?page='+page+'&rows='+rows, searchEntity);
    				}
    			});